Azure-Samples/Cognitive-Services-Voice-Assistant
# Move TTS Response to Custom Commands App in Hospitality

## Purpose

To demonstrate CC's functionality, we now generate the TTS response for commands within the CC interaction/completion rules, as opposed to in the Azure Function. This is done using both simple responses and lg templates.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

Make sure the Hospitality demo can still be deployed and function properly via the script/instructions.  No new hindrances should appear because of this change. The app should still function as it did before.

## What to Check
Verify that the following are valid
* ...

## Other Information
